namespace_registry/registry_db.go

```diff
@@ -227,15 +229,35 @@ func getPrefixJwksById(id int) (jwk.Set, error) {
 	return set, nil
 }
 
-func dbGetPrefixJwks(prefix string) (*jwk.Set, error) {
-	jwksQuery := `SELECT pubkey FROM namespace WHERE prefix = ?`
+func dbGetPrefixJwks(prefix string, adminApproval bool) (*jwk.Set, error) {
```

Comment:
Can you remove the adminApproval bool from the function signature since we always check if it's a cache anyway?